code/addons/vitest/src/postinstall.ts

```diff
@@ -121,66 +122,62 @@ export default async function postInstall(options: PostinstallOptions) {
       await writeFile(
         browserWorkspaceFile,
         dedent`
-        import { defineWorkspace } from 'vitest/config';
-        import { storybookTest } from '@storybook/experimental-addon-vitest/plugin';
-        ${vitestInfo.frameworkPluginImport ? vitestInfo.frameworkPluginImport + '\n' : ''}
-        export default defineWorkspace([
-          '${relative(dirname(browserWorkspaceFile), rootConfig)}',
-          {
-            plugins: [
-              storybookTest(),${vitestInfo.frameworkPluginCall ? '\n' + vitestInfo.frameworkPluginCall : ''}
-            ],
-            test: {
-              include: ['**/*.stories.?(m)[jt]s?(x)'],
-              browser: {
-                enabled: true,
-                name: 'chromium',
-                provider: 'playwright',
-                headless: true,
+          import { defineWorkspace } from 'vitest/config';
+          import { storybookTest } from '@storybook/experimental-addon-vitest/plugin';
+          ${vitestInfo.frameworkPluginImport ? vitestInfo.frameworkPluginImport + '\n' : ''}
+          export default defineWorkspace([
+            '${relative(dirname(browserWorkspaceFile), rootConfig)}',
+            {
+              extends: '${viteConfig || ''}',
```

Comment:
If there is no viteConfig it now extends an empty string.
But in that case, I think we should just not extend anything

Reply:
Also we want to make this a relative path, similar as the one above:
`'${relative(dirname(browserWorkspaceFile), rootConfig)}',`